diplomacy/web/src/gui/pages/content_game.jsx

```diff
@@ -3169,24 +3190,41 @@ export class ContentGame extends React.Component {
                     </div>
                 )}
                 {suggestionType !== null && (suggestionType & UTILS.SuggestionType.MOVE) === UTILS.SuggestionType.MOVE && (
-                    <ChatContainer
-                        style={{
-                            display: "flex",
-                            border: "1px solid black",
-                            boxSizing: "border-box",
-                        }}
-                    >
-                        <ConversationHeader>
-                            <ConversationHeader.Content
-                                userName={`Moves Advice for ${engine.phase}`}
-                            />
-                        </ConversationHeader>
-
-                        <MessageList>
-                            {fullSuggestionComponent}
-                            {partialSuggestionComponent}
-                        </MessageList>
-                    </ChatContainer>
+                    <div>
+                    <div>
+                        <Button
+                            title={"Get ally-based advice"}
+                            color={"primary"}
+                            onClick={() => {
+                                if (latestMoveSuggestionFull) {
+                                    this.handleRecipientAnnotation(latestMoveSuggestionFull.time_sent, "replace");
+                                }
+                                
+                                this.sendMessage(
+                                    engine.client,
+                                    "GLOBAL",
+                                    `${JSON.stringify(this.state.stances)}`,
+                                    null,
+                                    null
+                                );
```

Comment:
Are you sending stance changes via messages? If so, you should probably have a specific message type and a specific player it is targeted at.

Reply:
This is a player requesting a set of advice, based on their ally information, so it doesn't have a protagonist. I've updated relevant parts for adding `move_advice_request` for the type property.

Reply:
Isn't the player making the request the protagonist? How does an advisor know what player to respond to if they aren't informed about it?

Reply:
the player is represented as `engine.client` which is the sender of the message.

Reply:
As long as the advisor is looking at the sender, then it's fine for me!